custom/intrahealth/reports/utils.py

```diff
@@ -22,6 +25,38 @@ def get_localized_months():
         return [(_(calendar.month_name[i])).title() for i in range(1, 13)]
 
 
+class MonthYearMixin(object):
```

Comment:
Couldn't you import from https://github.com/dimagi/commcare-hq/blob/81565c979570c74e40540ceabd668b0b666e0bab/corehq/apps/reports/standard/__init__.py#L252 ?

Reply:
I have just cleaned it up to inherit from it as You are correct, it is mainly that with a small change that allows me to test it properly.